src/Apps/Settings/SettingsApp.tsx

```diff
@@ -63,9 +63,9 @@ const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
       <MetaTags title="Settings | Artsy" />
 
       <>
-        <Breadcrumbs>
+        <Breadcrumbs py={[2, 1]}>
```

Comment:
Another re-invention of the top context bar. This isn't displaying a breadcrumb trail either.